backend/templates/terms-of-use.html

```diff
@@ -0,0 +1,72 @@
+
+<p>Die Eingabe und Übermittlung der persönlichen Daten durch die Nutzer erfolgt online über die von match4healthcare vorbereiteten Eingabemasken oder über andere ausdrücklich von match4healthcare im Einzelfall vorgesehene Übermittlungswege. Für die Vollständigkeit und Richtigkeit sämtlicher vom Nutzer in die Eingabemaske eingegebenen oder sonst übermittelten Daten ist ausschließlich der Nutzer selbst verantwortlich. Sofern Nutzer falsche, unwahre oder unzureichende Angaben machen, ist match4healthcare berechtigt, die Anfrage unbearbeitet zu lassen. Für den Fall der Übermittlung von vorsätzlich falschen oder unwahren persönlichen Daten durch die Nutzer behält sich match4healthcare vor, diese von der Inanspruchnahme der angebotenen bzw. zur Verfügung gestellten Leistungen auszuschließen und Ersatz für dadurch eingetretene Schäden zu verlangen. </p>
+
+<h4>Wie haftet match4healthcare?</h4>
+
+<p>match4healthcare erstellt, wartet und unterhält die Online-Plattform, agiert jedoch nicht selbst aktiv als Vermittler zwischen den Nutzern. Es besteht kein Anspruch auf Nutzung der Plattform match4healthcare. Wartungsarbeiten, technische Ausfälle oder Fehler können dazu führen, dass sie nicht verfügbar ist.</p>
+
+<p>Darüber hinaus kann trotz aller Bemühungen nicht hundertprozentig ausgeschlossen werden, dass durch die Nutzung der Angebote Schäden entstehen, etwa wenn Dritte sie trotz aller Sicherheitsvorrichtungen zur Verbreitung von Viren oder Ähnlichem zu missbrauchen versuchen. Für Schäden, die aus der Nutzung der Angebote entstehen, übernehmen wir daher – auch für unsere gesetzlichen Vertreter, leitenden Angestellten und Erfüllungsgehilfen – grundsätzlich keine Haftung, außer bei vorsätzlichem oder grob fahrlässigem Handeln (bei Verletzung des Lebens, des Körpers oder der Gesundheit schließen wir die Haftung auch für einfache Fahrlässigkeit nicht kategorisch aus).</p>
+
+<p>Wir übernehmen ausdrücklich keine Gewähr für Richtigkeit, Vollständigkeit, Verlässlichkeit, Geeignetheit und Aktualität sowie Brauchbarkeit der Inhalte. Diese dienen ausschließlich der allgemeinen Information. Hinweise, Empfehlungen und Auskünfte sind unverbindlich. Jegliche Verwendung der angebotenen Inhalte geschieht auf eigenes Risiko. Insbesondere nimmt match4healthcare keine Prüfung der vom Nutzer angegebenen Qualifikationen vor.</p>
+
+<h4>Wie schützt match4healthcare Daten?</h4>
+
+<p>Zur Gewährleistung der Datenminimierung gibt match4healthcare keine Namen und Kontaktdaten von Helfenden an die Hilfesuchenden weiter. Die unmittelbare Kontaktaufnahme erfolgt stets durch den Helfenden. Die Darstellung der Helfenden findet auf der Helferkarte lediglich in Gestalt der Anzahl der Helfenden in einem bestimmten Postleitzahlenbereich statt. In den Hilfesuchenden in ihrem Nutzerkonto zur Verfügung gestellten Listen werden darüber hinaus Qualifikationen, die Verfügbarkeit und mitgeteilt, ob der Helfende zu eine Unterkunft wünscht und eine Bezahlung benötigt.</p>
```

Comment:
```suggestion
<p>Zur Gewährleistung der Datenminimierung gibt match4healthcare keine Namen und Kontaktdaten von Helfenden an die Hilfesuchenden weiter. Die unmittelbare Kontaktaufnahme erfolgt stets durch den Helfenden. Die Darstellung der Helfenden findet auf der Helferkarte lediglich in Gestalt der Anzahl der Helfenden in einem bestimmten Postleitzahlenbereich statt. In den Hilfesuchenden zur Verfügung gestellten Listen über potentielle Helfende werden darüber hinaus Qualifikationen, Verfügbarkeit sowie ein eventueller Wunsch nach Unterkunft und Bezahlung der Hilfe bietenden mitgeteilt.</p>
```
The last sentence was a bit hard to understand, maybe this makes it clearer

Reply:
Hm, finds dann fast verständlicher wenns "In den *den* Hilfesuchenden" zur Verfügung [...]" ist :see_no_evil: Ist mir persönlich aber egal.

Reply:
Ist "Zur Gewährleistung der Datenminimierung gibt match4healthcare keine Namen und Kontaktdaten von Helfenden an die Hilfesuchenden weiter." wirklich korrekt, wenn ein Helfender direkt einen Hilfesuchenden anschreibt?

Reply:
> Hm, finds dann fast verständlicher wenns "In den _den_ Hilfesuchenden" zur Verfügung [...]" ist 🙈 Ist mir persönlich aber egal.

Gerne auch das:) Ich bin nur bin nur über den Satz "zu eine Unterkunft wünscht" gestolpert und hab ihn nicht ganz verstanden. Ich glaube "zu eine[r] Unterkunft wünscht" passt auch.

Reply:
```suggestion
<p>Zur Gewährleistung der Datenminimierung gibt match4healthcare keine Namen und Kontaktdaten von Helfenden an die Hilfesuchenden weiter. Die unmittelbare Kontaktaufnahme erfolgt stets durch den Helfenden. Die Darstellung der Helfenden findet auf der Helferkarte lediglich in Gestalt der Anzahl der Helfenden in einem bestimmten Postleitzahlenbereich statt. In den den Hilfesuchenden zur Verfügung gestellten Listen über potentielle Helfende werden darüber hinaus Qualifikationen, Verfügbarkeit sowie ein eventueller Wunsch nach Unterkunft und Bezahlung der Hilfebietenden mitgeteilt.</p>
```